src/ORM/Connect/TempDatabase.php

```diff
@@ -195,13 +203,16 @@ public function build()
         set_error_handler($oldErrorHandler);
 
         // Ensure test db is killed on exit
-        register_shutdown_function(function () {
-            try {
-                $this->kill();
-            } catch (Exception $ex) {
-                // An exception thrown while trying to remove a test database shouldn't fail a build, ignore
-            }
-        });
+        $teardownOnExit = Config::inst()->get(static::class, 'teardown_on_exit');
```

Comment:
I think I'd rather see `$this->config()->get('teardown_on_exit')`, thoughts?

Reply:
ah yes - that's a bit more consistent + less verbose 👍 done.

Reply:
My bad, this class doesn't `use Configurable`. It does have config props on it, but rather than adding the trait, maybe revert back to what you had before. Sorry!

Reply:
haha oops - thanks travis :)